firmware/lm32/heartbeat.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <generated/csr.h>
+#include <generated/mem.h>
+#include <hw/flags.h>
+#include <system.h>
+#include <time.h>
+
+#define HDMI_IN0_FRAMEBUFFERS_BASE 	0x01000000
+#define HDMI_IN1_FRAMEBUFFERS_BASE 	0x02000000
+#define PATTERN_FRAMEBUFFER_BASE 	0x03000000
+
+#define YCBCR422_RED    0x544cff4c
+
+void heartbeat(int h_active, int m_active, )
```

Comment:
Removed for now. But need to have one more variable passed through which choose which of the input streams has to be changed with a 4x4 pixel at right bottom.